service/history/transferQueueStandbyProcessor.go

```diff
@@ -427,34 +435,50 @@ func (t *transferQueueStandbyProcessorImpl) processStartChildExecution(transferT
 func (t *transferQueueStandbyProcessorImpl) processRecordWorkflowStarted(transferTask *persistence.TransferTaskInfo) error {
 	processTaskIfClosed := false
 
+	return t.processTransfer(processTaskIfClosed, transferTask, func(msBuilder mutableState) error {
+		return t.processRecordWorkflowStartedOrUpsertHelper(transferTask, msBuilder, true)
+	}, standbyTaskPostActionNoOp)
+}
+
+func (t *transferQueueStandbyProcessorImpl) processUpsertWorkflowSearchAttributes(transferTask *persistence.TransferTaskInfo) error {
+	processTaskIfClosed := false
+
+	return t.processTransfer(processTaskIfClosed, transferTask, func(msBuilder mutableState) error {
+		return t.processRecordWorkflowStartedOrUpsertHelper(transferTask, msBuilder, false)
+	}, standbyTaskPostActionNoOp)
+}
+
+func (t *transferQueueStandbyProcessorImpl) processRecordWorkflowStartedOrUpsertHelper(transferTask *persistence.TransferTaskInfo, msBuilder mutableState, isRecordStart bool) error {
+	ok, err := verifyTaskVersion(t.shard, t.logger, transferTask.DomainID, msBuilder.GetStartVersion(), transferTask.Version, transferTask)
```

Comment:
what does this TransferTaskTypeUpsertWorkflowSearchAttributes task do?
if only sync-ing search attributes to ES or Cass, than you should not version control

Reply:
same for active transfer task processing

Reply:
Ok changed.